packages/flutter_driver/lib/src/error.dart

```diff
@@ -47,16 +52,39 @@ void _log(LogLevel level, String loggerName, Object message) {
 final Stream<LogRecord> flutterDriverLog = _logger.stream;
 
 /// Severity of a log entry.
-enum LogLevel { trace, info, warning, error, critical }
+enum LogLevel {
+  /// Messages used to supplement the higher-level messages with more details.
+  ///
+  /// This will likely produce a lot of output.
+  trace,
+
+  /// Informational messages that do not indicate a problem.
+  info,
+
+  /// A potential problem.
+  warning,
+
+  /// A certain problem but the program will attempt to continue.
+  error,
+
+  /// A critical problem; the program will attempt to quit immediately.
+  critical,
+}
 
 /// A log entry.
 class LogRecord {
   const LogRecord._(this.level, this.loggerName, this.message);
 
+  /// The severity of the log record.
   final LogLevel level;
+
+  /// The name of the logger that logged the message.
   final String loggerName;
+
+  /// The log message.
```

Comment:
Are there conventions for how to format this string? e.g. capitalisation, punctuation